src/PaymentMethods/Voucher.php

```diff
@@ -68,6 +68,25 @@ public function getFormFields($generalFormFields): array
         return array_merge($generalFormFields, $paymentMethodFormFieds);
     }
 
+    /**
+     * Retrieve the default category saved in db option
+     *
+     * @return string
+     */
+    public function voucherDefaultCategory():string
+    {
+        $mealvoucherSettings = get_option(
+            'mollie_wc_gateway_voucher_settings'
+        );
+        if(!$mealvoucherSettings){
+            $mealvoucherSettings = get_option(
+                'mollie_wc_gateway_mealvoucher_settings'
+            );
+        }
+
+        return $mealvoucherSettings? $mealvoucherSettings['mealvoucher_category_default']:Voucher::NO_CATEGORY;
```

Comment:
Missing spaces near `?`, `:`.

I thought there was [`phpcs`](https://github.com/mollie/WooCommerce/blob/develop/phpcs.xml.dist) in this project. :thinking: